dispatcher/dispatcher.go

```diff
@@ -427,6 +431,10 @@ func (d *IotxDispatcher) HandleBroadcast(ctx context.Context, chainID uint32, pe
 		d.dispatchAction(ctx, chainID, message)
 	case *iotextypes.Block:
 		d.dispatchBlock(ctx, chainID, peer, message)
+	case *iotextypes.ResponseNodeInfoMessage:
```

Comment:
`iotextypes.NodeInfoMessage`

Reply:
The related iotex-proto PR has been merged. `ResponseNodeInfoMessage` maybe also acceptable as another message is `RequestNodeInfoMessage`